[
    {
        "content": "<p><a href=\"https://github.com/jeffparsons\">jeffparsons</a> added the bug label to <a href=\"https://github.com/bytecodealliance/wasmtime/issues/8241\">Issue #8241</a>.</p>",
        "id": 429616318,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1711446273
    },
    {
        "content": "<p>jeffparsons opened <a href=\"https://github.com/bytecodealliance/wasmtime/issues/8241\">issue #8241</a>:</p>\n<blockquote>\n<p>On Wasmtime 18, I had <code>wit/deps/cli</code>, <code>wit/deps/clocks</code>, etc. as symlinks to vendored WASI wit files.</p>\n<p>After upgrading to Wasmtime 19, this same structure results in the error:</p>\n<blockquote>\n<p>failed to resolve directory while parsing WIT for path</p>\n</blockquote>\n<p>Replacing the symlinks with copies of the target directory resolves the issue.</p>\n<p>I'm not sure if this will be regarded as a bug, but I don't see why symlinks shouldn't be supported here. I guess it was just an accidental regression with recent bindgen changes? <span aria-label=\"man shrugging\" class=\"emoji emoji-1f937-200d-2642\" role=\"img\" title=\"man shrugging\">:man_shrugging:</span> </p>\n</blockquote>",
        "id": 429616321,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1711446274
    },
    {
        "content": "<p>alexcrichton <a href=\"https://github.com/bytecodealliance/wasmtime/issues/8241#issuecomment-2020557837\">commented</a> on <a href=\"https://github.com/bytecodealliance/wasmtime/issues/8241\">issue #8241</a>:</p>\n<blockquote>\n<p>Thanks for the report! I think that this is fixed by <a href=\"https://github.com/bytecodealliance/wasm-tools/pull/1437\">https://github.com/bytecodealliance/wasm-tools/pull/1437</a>, so I'll get new versions of those crates published so we can pull them in here.</p>\n</blockquote>",
        "id": 429665513,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1711462705
    },
    {
        "content": "<p>alexcrichton closed <a href=\"https://github.com/bytecodealliance/wasmtime/issues/8241\">issue #8241</a>:</p>\n<blockquote>\n<p>On Wasmtime 18, I had <code>wit/deps/cli</code>, <code>wit/deps/clocks</code>, etc. as symlinks to vendored WASI wit files.</p>\n<p>After upgrading to Wasmtime 19, this same structure results in the error:</p>\n<blockquote>\n<p>failed to resolve directory while parsing WIT for path</p>\n</blockquote>\n<p>Replacing the symlinks with copies of the target directory resolves the issue.</p>\n<p>I'm not sure if this will be regarded as a bug, but I don't see why symlinks shouldn't be supported here. I guess it was just an accidental regression with recent bindgen changes? <span aria-label=\"man shrugging\" class=\"emoji emoji-1f937-200d-2642\" role=\"img\" title=\"man shrugging\">:man_shrugging:</span> </p>\n</blockquote>",
        "id": 429707295,
        "sender_full_name": "Wasmtime GitHub notifications bot",
        "timestamp": 1711474749
    }
]